docs/reading-data.qmd

```diff
@@ -81,9 +81,14 @@ Depending on the provider, you will have to set up the appropriate credentialing
 ```python
 import polars as pl
 
-source = "s3://bucket/yourfilehere.parquet"
-
-df = pl.read_parquet(source)
+DATA_BUCKET = "s3://my-bucket/data/"
+STORAGE_OPTIONS = {
+    "aws_access_key_id": "<secret>",
+    "aws_secret_access_key": "<secret>",
+    "aws_region": "us-east-1",
+}
```

Comment:
Why are we including `STORAGE_OPTIONS` if we aren't using it anywhere? FWIW, I feel like since we link to Polars docs, we don't necessarily need to get into auth details